app/src/test/java/com/eventyay/organizer/core/sessions/create/CreateSessionViewModelTest.java

```diff
@@ -0,0 +1,237 @@
+package com.eventyay.organizer.core.sessions.create;
+
+import android.arch.core.executor.testing.InstantTaskExecutorRule;
+import android.arch.lifecycle.Observer;
+import com.eventyay.organizer.common.ContextManager;
+import com.eventyay.organizer.core.session.create.CreateSessionViewModel;
+import com.eventyay.organizer.data.event.Event;
+import com.eventyay.organizer.data.session.Session;
+import com.eventyay.organizer.data.session.SessionRepository;
+import com.eventyay.organizer.utils.DateUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.InOrder;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.threeten.bp.LocalDateTime;
+
+import io.reactivex.Observable;
+import io.reactivex.android.plugins.RxAndroidPlugins;
+import io.reactivex.plugins.RxJavaPlugins;
+import io.reactivex.schedulers.Schedulers;
+
+import static org.junit.Assert.assertNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(JUnit4.class)
+public class CreateSessionViewModelTest {
+
+    @Rule
+    public MockitoRule mockitoRule = MockitoJUnit.rule();
+    @Rule
+    public TestRule rule = new InstantTaskExecutorRule();
+    @Mock
+    private SessionRepository sessionRepository;
+    @Mock
+    private Event event;
```

Comment:
`event` should have a valid `start date`, `end date` and `timezone`.

Reply:
Yes, but I didn't change the way we are getting start date, end date and time zone. It is same as earlier, It was working for presenterTests but not for viewModelTests. Pls guide

Reply:
Where are you getting start date, end date and time zone in this file? The event mock will return null when getters for timezone etc are called.